js/client/modules/@arangodb/process-utils.js

```diff
@@ -1479,6 +1483,17 @@ function checkClusterAlive(options, instanceInfo, addArgs) {
       throw new Error('cluster startup timed out after 10 minutes!');
     }
   }
+
+  if (options.cluster) {
```

Comment:
Any suite that has intentional arangod suspending will face false status for such inspection.